spec/views/students/show.html.erb_spec.rb

```diff
@@ -1,5 +1,13 @@
 require 'rails_helper'
 
-RSpec.describe "students/show.html.erb", type: :view do
-  pending "add some examples to (or delete) #{__FILE__}"
+RSpec.describe "students/show", type: :view do
+  let(:student) { build_stubbed :student }
+  before do
+    render template: subject, locals: {
+      student: student.decorate
+    }
+  end
+  it "shows the student" do
+    expect(rendered).to match(/Stats/)
```

Comment:
yep, it's because we have so many things that a student might not have filled out.